driver/catalogue.c

```diff
@@ -605,9 +605,15 @@ SQLRETURN TEST_API update_varchar_defs(esodbc_stmt_st *stmt)
 	long row_cnt;
 	wstr_st dest = {0};
 	size_t pos;
+	SQLULEN max_length;
 	cstr_st u8mb;
 	wstr_st *lim = &HDRH(stmt)->dbc->varchar_limit_str;
 
+	/* save and reset SQL_ATTR_MAX_LENGTH attribute, it'll interfere with
+	 * reading lenght of avail data with SQLGetData() otherwise. */
```

Comment:
typo: lenght -> length

Reply:
I'll have to install a precommit check for this one. Thanks, fixed them all.